apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  Text,
+  IconButton,
+  NotificationMarker,
+  NotificationMarkerProps,
+} from '@itwin/itwinui-react';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  component: NotificationMarker,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    type: {
+      control: 'radio',
+      options: ['primary', 'positive', 'warning', 'negative'],
+    },
+  },
+  title: 'Core/NotificationMarker',
+} as Meta<NotificationMarkerProps>;
+
+export const Basic: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <IconButton styleType='borderless'>
+      <NotificationMarker {...args}>{args.children}</NotificationMarker>
+    </IconButton>
+  );
+};
+
+Basic.args = {
+  type: 'primary',
+  urgent: false,
+  children: <SvgNotification />,
+};
+
+Basic.argTypes = {
+  children: {
+    control: false,
+  },
+};
+
+export const CustomChild: Story<NotificationMarkerProps> = (args) => {
```

Comment:
I was trying to have a story to show that we can pass anything as a child instead of just SVGs like `<SvgNotification />`.

Maybe `CustomChild` is not an accurate name. What name do you recommend for such a story?

Reply:
I was trying to have a story to show that we can pass anything as a child instead of just SVGs like `<SvgNotification />`.

Maybe `CustomChild` is not an accurate name. What name do you recommend for such a story?

Reply:
Maybe you could just put one text example into Basic? And replace the Custom example with a Pulsing example. Because in both Basic and Custom examples, the notification marker itself is basic.

Reply:
**[DONE]** Ohh, nice idea, thanks. I changed `CustomChild` story to `Pulsing`.

`Basic` now shows one icon button and one text. `Pulsing` shows four texts (for each status) with pulsing notifications

Reply:
**UPDATE**: I now created another story called `Status`. So now, `Basic` is same as before, `Pulsing` shows one icon and one bold text (similar to Basic), and `Status` shows all four statuses for icons and texts at once